src/main/java/org/opentripplanner/raptor/api/request/MultiCriteriaRequest.java

```diff
@@ -102,8 +93,8 @@ public List<PassThroughPoint> passThroughPoints() {
    * is replaced by {@link #relaxC1()}. This parameter is ignored if {@link #relaxC1()} exist.
    */
   @Deprecated
-  public Optional<Double> relaxCostAtDestination() {
-    return Optional.ofNullable(relaxCostAtDestination);
+  public Double relaxCostAtDestination() {
```

Comment:
Can this return `null`? If so then please add `@Nullable`. If not, then use a primitive `double`.

Reply:
I added `@Nullable`. I am removing this soon. I just want to provide transit-group-priority as an alternative first. I have not checked lately, but we had on client using it.